### PR TITLE
Flair purchase script

### PR DESCRIPTION
**Pull request by Pongch**
this script should:
1. filter through all paged transactions sent to a specified burn address 
2. loop through different price && metadata for each flair input
3. fetch all the user's existing flairs
4. append the flairs if the users don't have them

### CODE REVIEW

File: scripts/src/index.js [hunk 1]
```diff
@@ -34,27 +33,29 @@ const r = new snoowrap({
...
+        getFlair('flair_soon', ':soon:', 10000)
+      )
+      console.log(flairArrays)
+      const update = await flair.setFlairs(
+        flairArrays,
+        r.getSubreddit(subreddit)
+      )
+      console.log(update)
+    })} catch(err) {
     console.log(err)
```
Comment (nicholasmueller) [inline scripts/src/index.js:59]: should we log to sentry? potentially noisy but maybe better than waiting for a user to report support

File: scripts/src/flair.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+  parseThreadJSON: function (json) {
+    const rawComments = get(json, '[1].data.children', []);
+
+    let userAddressMap = [];
+
+    // sanitize and filter comments for addresses
+    for (const comment of rawComments) {
+      const rawText = comment.data.body;
+      const words = rawText.trim().split(' ');
```
Comment (nicholasmueller) [inline scripts/src/flair.js:40]: added a fix here in the extension for not parsing the newlines so you just have to add
```
const words = rawText.trim().replace(/\n/g, ' ').split(' ');
```

File: scripts/src/index.js (hunk 1 above)
Comment (Pongch) [reply scripts/src/index.js:59]: yup, I'm planning to

Comment (jbunce) [reply scripts/src/index.js:59]: +1 @nicholasmueller

**Pull request merged**